Tools/python/ROOTTools.py

```diff
@@ -1984,7 +1986,7 @@ def zzz(h):
             if rs:
                 min_r = min(min_r, min(rs))
                 max_r = max(max_r, max(rs))
-        res_y_range = min_r*(1-res_y_range), max_r*(1+res_y_range)
+        res_y_range = max(min_r*(1-res_y_range),-2.5), min(max_r*(1+res_y_range),2.5)
```

Comment:
This sets the ratio pad to only use a y-axis range of [-2.5, +2.5] at the largest, right? If so, I think it's probably a good idea! (since ratio pads that go much beyond that are usually very hard to use)

I imagine we could have a use-case in the future where having this be configurable could be useful, but let's not worry about it now.

Reply:
Yes I found sometimes it goes too far beyond so changed that.